apps/studio/src/lib/s3.ts

```diff
@@ -12,10 +12,14 @@ import { getSignedUrl } from "@aws-sdk/s3-request-presigner"
 
 import { env } from "~/env.mjs"
 
-const { NEXT_PUBLIC_S3_REGION } = env
+const { NEXT_PUBLIC_S3_REGION, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY } = env
 
 export const storage = new S3Client({
   region: NEXT_PUBLIC_S3_REGION,
+  credentials: {
+    accessKeyId: AWS_ACCESS_KEY_ID ?? "",
```

Comment:
Nit: Should we not even pass in the credentials sub-object if it does not exist? Empty strings feel iffy over undefined.